service/service-mock/service-mock-soap/src/main/java/com/castlemock/service/mock/soap/project/ImportSoapResourceService.java

```diff
@@ -50,23 +52,24 @@
         final SoapResource soapResource = input.getResource();
         final String raw = input.getRaw();
 
+        XPath xPath = XPathFactory.newInstance().newXPath();
+        String path = "/definitions/portType/@name";
+
+        String portName = null;
+        try {
+            portName = xPath.compile(path).evaluate(raw);
+        } catch (XPathExpressionException e) {
+            e.printStackTrace();
```

Comment:
Should we fallback to the soapResourceName here instead of just printing the stacktrace. This could cause issues further down.